porting/introduction/index.rst

```diff
@@ -5,9 +5,9 @@ Porting is the process of making Ubuntu Touch run on new hardware, i.e. on devic
 
 On the vast majority of commercially available devices crucial parts of their systems are not open source. Furthermore, these proprietary parts are specific, not only to the manufacturer, but usually also to the individual model. 
 
-Porting Ubuntu Touch involves building it in such a way as to integrate it with these proprietary components on a device so as to enable it to "talk to" the device hardware. 
+Porting Ubuntu Touch involves building it in such a way so that it would be able  to integrate with these proprietary components on a device so as to enable it to "talk to" the device hardware. 
 
-Before attempting to port Ubuntu Touch to a new device, there are a few things to understand and consider, some research to be done, and a suitable build environment needs to be set up. This section addresses these points.
+Before attempting to port Ubuntu Touch to a new device, there are a few things to understand and consider, some research to be done(like knowing your device hardware), and a suitable build environment needs to be set up. This section addresses these points.
```

Comment:
mhm .. I'm not so sure this really improves the text. porting is a long chapter, so I think it's ok to have a short, easy to ready intro without too many detail. they all come later ... what do you think? (at the very least there is a space missing in front of the bracket)